contracts/Core/StakeManager.sol

```diff
@@ -272,7 +330,7 @@ contract StakeManager is Utils, ACL, StakeStorage {
         if (epochs < 10) {
             return(stakeValue);
         }
-        // penalty =( epochs -1)*stakeValue*penNum/penDiv
+
         uint256 penalty = (epochs-(1))*((stakeValue*(Constants.penaltyNotRevealNum()))/(
```

Comment:
Multiplication should happen first before division to avoid situations where the result is 0 due to integer divison

Reply:
This was commented, if we decide to keep it, then I will fix it. @hrishikeshio